Packs/MicrosoftAdvancedThreatAnalytics/Integrations/MicrosoftAdvancedThreatAnalytics/MicrosoftAdvancedThreatAnalytics.py

```diff
@@ -0,0 +1,348 @@
+import traceback
+from typing import Any, Dict, List, Tuple, Union
+
+import urllib3
+from dateparser import parse
+from pytz import utc
+from requests_ntlm import HttpNtlmAuth
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+urllib3.disable_warnings()
+
+SEVERITY_TRANSLATION = {
+    'Low': 1,
+    'Medium': 2,
+    'High': 3
+}
+
+
+class Client(BaseClient):
+    def get_suspicious_activity_request(self, suspicious_activity_id: str = '') -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/suspiciousActivities/{suspicious_activity_id}'
+        )
+
+    def get_suspicious_activity_details_request(self, suspicious_activity_id: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/suspiciousActivities/{suspicious_activity_id}/details'
+        )
+
+    def update_suspicious_activity_status_request(self,
+                                                  suspicious_activity_id: str,
+                                                  suspicious_activity_status: str
+                                                  ) -> Dict[str, Any]:
+        body = {
+            'Status': suspicious_activity_status
+        }
+
+        return self._http_request(
+            method='POST',
+            url_suffix=f'/suspiciousActivities/{suspicious_activity_id}',
+            json_data=body,
+            ok_codes=(204,),
+            resp_type='text'
+        )
+
+    def delete_suspicious_activity_request(self, suspicious_activity_id: str) -> Dict[str, Any]:
+        body = {
+            'shouldDeleteSametype': False
+        }
+
+        params = {
+            'shouldDeleteSameType': 'false'
+        }
+
+        return self._http_request(
+            method='DELETE',
+            url_suffix=f'/suspiciousActivities/{suspicious_activity_id}',
+            json_data=body,
+            params=params,
+            ok_codes=(204,),
+            resp_type='text'
+        )
+
+    def get_monitoring_alert_request(self) -> Dict[str, Any]:
```

Comment:
no why to add the filters to the request?
```suggestion
    def get_monitoring_alerts_request(self) -> Dict[str, Any]:
```